tests/system_tests_two_routers.py

```diff
@@ -470,19 +497,17 @@ def on_message(self, event):
                                 'adminStatus': 'deleted'}
                             request.reply_to = self.mgmt_receiver_1.remote_source.address
                             self.mgmt_sender.send(request)
+                            self.n_sent += 1
         elif event.receiver == self.mgmt_receiver_1:
+            self.n_received += 1
             if event.message.properties['statusDescription'] == 'OK' and event.message.body['adminStatus'] == 'deleted':
```

Comment:
I would argue if this response is not OK and deleted the test should immediately fail.